src/interface/decompositions.jl

```diff
@@ -151,6 +151,19 @@ a general matrix using the Jacobi algorithm.
 """
 @algdef CUSOLVER_Jacobi
 
+"""
+    CUSOLVER_Randomized(; p, niters)
```

Comment:
This misses `k` as a keyword argument, as well as an explanation of `p` and `niters`.